docs/guides/accessibility.md

```diff
@@ -27,84 +39,207 @@ The `@angular-eslint` repo contains a number of linting rules that can help enfo
 Most of the accessibility rules that are enabled in the Intershop PWA are contained in the plugin `@angular-eslint/template/accessibility` that is configured in the `.eslintrc.json` file of the project.
 To check whether the rules are followed in your custom code or not, run `npm run lint`.
 
-## General
+## Check the Accessibility of a Page
```

Comment:
The text above doesn't fit to the new content of this documentation. (Most of the ... npm run lint)
- Start with a general motivation, the 1st sentence is good, maybe you can mention the guidelines ....
- move the lint part to the Eslint Rules section (only what is needed)